spring-cloud-gcp-secretmanager/src/test/java/com/google/cloud/spring/secretmanager/SecretManagerTemplateTests.java

```diff
@@ -181,6 +183,21 @@ void testAccessSecretString() {
     assertThat(result).isEqualTo("get after it.");
   }
 
+  @Test
+  void testAccessNonExistedSecretString() {
```

Comment:
It's easier to troubleshoot failing tests when each test only tests one scenario. Could you split this test into 2 -- one testing the throwing behavior, and the other testing the returning-null behavior?